### ISSUE TITLE
deps convert output to stdout

### ISSUE THREAD

**Issue by dbanty** [dephell/dephell#136]
It would be useful (at least to me) to be able to output the results of deps convert to wherever I want instead of it creating a file for me.  My specific use case is actually just a workaround while I wait for #129.

Currently my process looks like this to make a requirements.txt:
```bash
dephell deps convert # Creates a requirements.txt
echo --extra-index-url=$PRIVATE_URL > temp.txt
tail -n +2 requirements.txt >> temp.txt # Includes everything but the -i which doesn't work with --extra-index-url
rm requirements.txt
mv temp.txt requirements.txt
```

Obviously this would be nicer if I could do:
```bash
echo --extra-index-url=$PRIVATE_URL > requirements.txt
dephell deps convert | tail -n +2 >> requirements.txt
```

Not sure if there will be too many use cases after #129 is done, maybe for piping the output to pip for some reason?

**Comment by orsinium** [comment]
Extra indices is already implemented in #129. I'm not ready to merge it, because I'm not sure about some corner cases yet :)

Anyway, I thought about stdout output, that's good point. Let's do it.

**Comment by orsinium** [comment]
Realization is quite simple, btw. Just check that output file name is "stdout" and do `print(dumper.dumps(...))`

**Comment by dbanty** [comment]
Cool, I can give it a shot later tonight if you want.

**Comment by orsinium** [comment]
Closed in #140
